curation-migration-datasync/database/dataService.ts

```diff
@@ -83,8 +84,48 @@ export class DataService {
     }
   }
 
-  public async deleteScheduledItem() {
-    return;
+  public async deleteScheduledItem(curatedRecId: number) {
```

Comment:
this looks good to me, but given its a high risk service for for 2-3 months, its good to write an integration test for happy path alone , so our test can catch it if the code changes/bugs introduced somewhere in future PRs.

Reply:
sounds good, I did it